include/experimental/__p1673_bits/blas2_matrix_vector_solve.hpp

```diff
@@ -64,20 +64,21 @@ void trsv_upper_triangular_left_side(
 {
   constexpr bool explicit_diagonal =
     std::is_same_v<DiagonalStorage, explicit_diagonal_t>;
+  using size_type = typename extents<>::size_type;
 
-  const ptrdiff_t A_num_rows = A.extent(0);
-  const ptrdiff_t A_num_cols = A.extent(1);
+  const size_type A_num_rows = A.extent(0);
+  const size_type A_num_cols = A.extent(1);
 
   // One advantage of using signed index types is that you can write
```

Comment:
Alas, we're not using signed index types any more, so this code will loop forever (`i >= 0`, always).